scaleway/resource_rdb_instance_beta.go

```diff
@@ -234,13 +236,26 @@ func resourceScalewayRdbInstanceBetaUpdate(d *schema.ResourceData, m interface{}
 	if err != nil {
 		return err
 	}
-
+	var upgradeInstanceRequests []rdb.UpgradeInstanceRequest
```

Comment:
nit: we try to use `:=` everywhere.

```suggestion
	upgradeInstanceRequests := []rdb.UpgradeInstanceRequest(nil)
```